neuralprophet/forecaster.py

```diff
@@ -328,12 +328,7 @@ def __init__(
         # AR
         self.config_ar = configure.from_kwargs(configure.AR, kwargs)
         self.n_lags = self.config_ar.n_lags
-        if n_lags == 0 and n_forecasts > 1:
-            self.n_forecasts = 1
-            log.warning(
-                "Changing n_forecasts to 1. Without lags, the forecast can be "
-                "computed for any future time, independent of lagged values"
-            )
+        self.max_lags = self.n_lags
```

Comment:
should this not also check for covars?

Reply:
I guess not. We only check for covars when we fit the model as we cannot add_lagged_regressors when instantiating NeuralProphet()

Reply:
Yeah, makes sense. We should just make sure that this is not used before that function is called in fit.